### PR TITLE
DOCS-716 asses onboarding docs + extra nits

### PR DESCRIPTION
**Pull request by andf-viam**
Small tweaks to low-hanging fruit where discussion or further consideration is unlikely to be necessary. Kept most changes to within onboarding sections -- Viam in 3 Minutes, Try Viam, and the Installation Guide -- but my monospace efforts for `viam-server` ended up being corpus wide 😅 .

Happy for any corrections if I did something wrong -- thanks!!

### CODE REVIEW

File: docs/try-viam/rover-resources/_index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ description: Get your own Viam rover and set it up.
 ---
 
 {{< alert title="Tip" color="tip" >}}
-If you want a convenient mobile {{% glossary_tooltip term_id="base" text="base"%}} for a variety of robotics projects, you can [order your own Viam rover](https://www.viam.com/resources/rover) now.
+If you want a convenient mobile {{% glossary_tooltip term_id="base" text="base"%}} for a variety of robotics projects, you can now [order your own Viam rover](https://www.viam.com/resources/rover).
```
Comment (andf-viam) [inline docs/try-viam/rover-resources/_index.md:18]: Personal nit. "order" or "buy" followed by "now" is a sales exhortation. Either preceded by "now" is instead a "recently-made-available" advisement. Obviously feel free to disregard this if I am being too pedantic 😅 .

File: docs/try-viam/rover-resources/rover-tutorial.md [hunk 2]
```diff
@@ -152,7 +152,7 @@ This is the recommended order to assemble your rover:
 
 1. [Install Raspberry Pi OS on the microSD card.](#install-raspberry-pi-os)
 2. [Unscrew the top of the rover and screw the Pi to the base.](#attach-the-raspberry-pi-to-the-rover)
-3. [Conenct the components.](#connect-the-wires)
+3. [Connect the components.](#connect-the-wires)
```
Comment (sguequierre) [inline docs/try-viam/rover-resources/rover-tutorial.md:155]: nice

**Review by sguequierre — APPROVED** [review]
wonderful catches, woohoo! I don't know if any of these changes require @npentrel's approval but I'll go ahead and approve if you want to get these fixes in. 
Nice work!

File: docs/appendix/glossary/client-application.md [hunk 3]
```diff
@@ -8,5 +8,5 @@ aka:
 
 Client applications run business logic to operate your robot.
 
-You can run a client application on the same {{< glossary_tooltip term_id="part" text="part" >}} that runs the viam-server, or on a separate device.
+You can run a client application on the same {{< glossary_tooltip term_id="part" text="part" >}} that runs the `viam-server`, or on a separate device.
```
Comment (npentrel) [inline docs/appendix/glossary/client-application.md:11]:
```suggestion
You can run a client application on the same {{< glossary_tooltip term_id="part" text="part" >}} that runs `viam-server`, or on a separate device.
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**